.github/workflows/wheels-build.yml

```diff
@@ -0,0 +1,170 @@
+    runs-on: ubuntu-latest
+    outputs:
+      RAPIDS_EPOCH_TIMESTAMP: ${{ steps.get_epoch_time.outputs.RAPIDS_EPOCH_TIMESTAMP }}
+      PIP_EXTRA_INDEX_URL: ${{ steps.get_pip_extra_index_url.outputs.PIP_EXTRA_INDEX_URL }}
+
+    steps:
+    - name: Get epoch timestamp to use as nightly version
+      id: get_epoch_time
+      run: echo "RAPIDS_EPOCH_TIMESTAMP=$(date +%s)" >> "${GITHUB_OUTPUT}"
+    - name: Setup extra index URL for pip
+      id: get_pip_extra_index_url
+      run: echo "PIP_EXTRA_INDEX_URL=https://pypi.k8s.rapids.ai/simple" >> "${GITHUB_OUTPUT}"
+
+  compute-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      MATRIX: ${{ steps.compute-matrix.outputs.MATRIX }}
+    steps:
+      - name: Compute Build Matrix
+        id: compute-matrix
+        run: |
+          set -eo pipefail
+
+          export MATRIX="
+          - { ctk: '11.8.0', arch: 'amd64', python: '3.9', linux_ver: 'centos7' }
+          - { ctk: '11.8.0', arch: 'amd64', python: '3.10', linux_ver: 'centos7' }
+          - { ctk: '11.8.0', arch: 'arm64', python: '3.9',  linux_ver: 'ubuntu20.04' }
+          - { ctk: '11.8.0', arch: 'arm64', python: '3.10', linux_ver: 'ubuntu20.04' }
+          - { ctk: '12.0.1', arch: 'amd64', python: '3.9',  linux_ver: 'centos7' }
+          - { ctk: '12.0.1', arch: 'amd64', python: '3.10',  linux_ver: 'centos7' }
+          - { ctk: '12.0.1', arch: 'arm64', python: '3.9', linux_ver: 'ubuntu20.04' }
+          - { ctk: '12.0.1', arch: 'arm64', python: '3.10', linux_ver: 'ubuntu20.04' }
+          "
+
+          echo "MATRIX=$(
+            yq -n -o json 'env(MATRIX)' | \
+            jq -c '${{ inputs.matrix_filter }} | {include: .}' \
+          )" | tee --append "${GITHUB_OUTPUT}"
+
+  wheel-build:
+    name: cibuildwheel ${{ matrix.arch }} ${{ matrix.python }} ${{ matrix.ctk }}
+    needs: [setup-env-vars, compute-matrix]
+    strategy:
+      matrix: ${{ fromJSON(needs.compute-matrix.outputs.MATRIX) }}
+    runs-on: "linux-${{ matrix.arch }}-${{ inputs.node_type }}"
+    container:
+      image: "rapidsai/ci-wheel:cuda${{ matrix.ctk }}-${{ matrix.linux_ver }}-py${{ matrix.python }}"
+
+      env:
+        RAPIDS_BUILD_TYPE: ${{ inputs.build_type }}
+        RAPIDS_PY_VERSION: ${{ matrix.python }}
+        RAPIDS_EPOCH_TIMESTAMP: ${{ needs.setup-env-vars.outputs.RAPIDS_EPOCH_TIMESTAMP }}
+        PIP_EXTRA_INDEX_URL: ${{ needs.setup-env-vars.outputs.PIP_EXTRA_INDEX_URL }}
+
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ vars.AWS_ROLE_ARN }}
+          aws-region: ${{ vars.AWS_REGION }}
+          role-duration-seconds: 43200 # 12h
+      - name: checkout code repo
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ inputs.repo }}
+          ref: ${{ inputs.sha }}
+          fetch-depth: 0 # unshallow fetch for setuptools-scm
+          persist-credentials: false
+
+      - name: Standardize repository information
```

Comment:
Why can't we reuse `rapids-github-info` here? Is there some issue with whether or not the changes propagate to

Reply:
Yes we can, sorry I got confused because I picked it up from conda workflows